cowbird/monitoring/monitoring.py

```diff
@@ -48,6 +49,11 @@ def start(self) -> None:
             self.monitors[mon.path][mon.callback] = mon
             mon.start()
 
+        # Initialize FileSystem handler which must monitor the wpsoutputs folder on startup
+        filesystem_handler = HandlerFactory().get_handler("FileSystem")
+        if filesystem_handler:
+            filesystem_handler.start_wpsoutputs_monitoring(self)
```

Comment:
I think it is not needed as it will be done in the `get_handler()` method, returning None, if the handler is not defined in the config or if it has `active=False`.

See [here](https://github.com/Ouranosinc/cowbird/blob/DAC-570-mount-public-wpsoutputs/cowbird/handlers/handler_factory.py#L78-L82)